docs/pages/tutorial/eas/ios-development-build-for-devices.mdx

```diff
@@ -0,0 +1,229 @@
+sidebar_title: iOS development build for devices
+description: Learn how to configure a development build for iOS devices using EAS Build.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, let's create our development build for iOS devices with EAS Build.
+
+## Trigger a build
+
+Development builds for iOS devices are generated in the **.ipa** format, which is standard for iOS app installations.
+
+> **Note**: Apple Developer membership is required to create and install a development build on an iOS device.
+
+<Step label="1">
+
+## Register an iOS device with a provisioning profile
+
+To initiate development on an iOS device, we have to:
+
+- Register the device by creating a new [provisioning profile](/app-signing/app-credentials/#provisioning-profiles).
+- Download and install this profile onto the device.
+
+With EAS CLI, run the command to register a new Apple device:
+
+<Terminal cmd={['$ eas device:create']} />
+
+This command prompts us with the following questions:
+
+- **You're inside the project directory. Would you like to use the** **your-account-name** **account?** Press <kbd>Y</kbd>.
+- **Apple ID.** For this step, enter your Apple ID. It will then log in to our Apple Developer account. Follow the steps in the terminal window.
+- **How would you like to register your devices?** Select **Website** that generates a registration URL that can be opened on the iOS device.
+
+> **info** **Tip**: If you or your team have multiple devices, you can share the provisioning profile link with those devices for downloading and installing the profile.
+
+</Step>
+
+<Step label="2">
+
+## Download and install the provisioning profile
+
+To register an iOS device for development:
+
+- On a device's web browser, open the link provided in the previous step and tap the **Download Profile button**.
+
+<ImageSpotlight
+  alt="Download the provisioning profile"
+  src="/static/images/tutorial/eas/ios-ad-hoc-01.jpg"
+  style={{ maxWidth: 280 }}
+/>
+
+- Open the **Settings** app, which prompts us to register our device.
+
+<ImageSpotlight
+  alt="Install the provisioning profile"
+  src="/static/images/tutorial/eas/ios-ad-hoc-02.jpg"
+  style={{ maxWidth: 280 }}
+/>
+
+- Tap **Install** to register the iOS device.
+
+After the provisioning profile is installed, our device redirects us back to the web browser, displaying a success message indicating the completion of the process.
+
+<ImageSpotlight
+  alt="Provisioning profile installed successfully"
+  src="/static/images/tutorial/eas/ios-ad-hoc-03.jpg"
+  style={{ maxWidth: 280 }}
+/>
+
+</Step>
+
+<Step label="3">
+
+## Create a development build
```

Comment:
Same as for Android, it would be handy to refer back to the part of `eas.json` we're talking about.